### PR TITLE
Add bt support/external adapter

### PR DESCRIPTION
**Pull request by pupdum**
This enables the Bluez Bluetooth stack and hotplug support for USB Bluetooth hardware to the OT2 configuration. It also newly adds the [Bluepy](https://github.com/IanHarvey/bluepy) package to Buildroot and the OT2 defconfig.

This will extend the capabilities of the OT2 and allow for more communication options. 

These changes were implemented by adding the new package, enabling Bluez tools and utils packages, and adding new systemd units and a udev rule. I tried to make the rule generic enough that it could match with most if not all USB Bluetooth dongles. 

As far as testing, I built these changes and have used a Bluetooth Low Energy capable dongle on an OT2 to send and receive data. I have successfully tested the plug and play capabilities with one device before the udev rule was relocated to its own file. I currently do not have access to an OT2 so someone else might want to check if that change broke anything.

### CODE REVIEW

File: configs/ot2_defconfig [hunk 1]
```diff
@@ -43,12 +43,12 @@ BR2_PACKAGE_RPI_USERLAND=y
 BR2_PACKAGE_UBOOT_TOOLS=y
 BR2_PACKAGE_USBMOUNT=y
 BR2_PACKAGE_PYTHON3=y
-BR2_PACKAGE_PYTHON3_BZIP2=y
```
Comment (sfoster1) [inline configs/ot2_defconfig]: We want this still I think... savedefconfig keeps deleting it for some reason but for now let's manually re-add it

**Review by sfoster1 — CHANGES_REQUESTED** [review]
One little change in the defconfig, but otherwise looks good!

File: configs/ot2_defconfig (hunk 1 above)
Comment (pupdum) [reply configs/ot2_defconfig]: Ahh, yeah there do seem to be some unexpected changes. I'll fix that.

**Comment by pupdum** [comment]
Branch needs to be renamed.

**Pull request closed without merging**